code/modules/antagonists/gang/gang.dm

```diff
@@ -146,6 +146,42 @@
 						/obj/item/clothing/under/suit/henchmen,
 						/obj/item/toy/crayon/spraycan)
 
+/datum/antagonist/gang/yakuza
+	name = "Tojo Clan"
+	roundend_category = "Yakuza"
+	gang_name = "Tojo Clan"
+	gang_id = "YAK"
+	acceptable_clothes = list(/obj/item/clothing/head/soft/yellow,
+							/obj/item/clothing/under/costume/yakuza,
+							/obj/item/clothing/shoes/yakuza,
+							/obj/item/clothing/neck/scarf/yellow,
+							/obj/item/clothing/head/beanie/yellow,
+							/obj/item/clothing/mask/bandana/gold,
+							/obj/item/clothing/head/hardhat,
```

Comment:
Is it wise to reward a gang with basically free armour for their default points earning uniform?

Reply:
hard-hats are absolutely abysmal for armor but yeah I could make an unarmored one for this